.github/workflows/release-official.yml

```diff
@@ -32,4 +32,7 @@ jobs:
         with:
           context: "{{defaultContext}}:explorer/ui"
           push: true
-          tags: omniops/explorer-ui:main,omniops/explorer-ui:${{ steps.git_ref.outputs.short_sha }}
+          tags: |
+            omniops/explorer-ui:main
+            omniops/explorer-ui:${{ steps.git_ref.outputs.short_sha }}
+            omniops/explorer-ui:${GITHUB_REF##*/}  # This adds the tag (e.g., v0.1.8)
```

Comment:
Does this work with the docker/build-push-action? I thought that only `,` sperator works 🤔